src/models.py

```diff
@@ -29,8 +29,8 @@
 
 from .preprocess import parse_task_list_arg, get_tasks
 
-from .tasks.tasks import CCGTaggingTask, ClassificationTask, CoLATask, CoLAAnalysisTask, \
-    GroundedSWTask, GroundedTask, MultiNLIDiagnosticTask, PairClassificationTask, \
+from .tasks.tasks import CCGTaggingTask, ClassificationTask, CoLATask, GroundedSWTask, \
+    GroundedTask, MultiNLIDiagnosticTask, PairClassificationTask, \
```

Comment:
This is an enormous change, and most of it doesn't have any obvious connection to PRPN. Trim this down to something more focused and we can review that.

Reply:
For the most part, you should only need to change sections of code that were changed in the ON-LSTM PR.